### PR TITLE
Feature/added logger to the new project

### PR DESCRIPTION
**Pull request by milmilkat**
Added logger but created a separate MR for you to see if it actually makes sense or not

### CODE REVIEW

File: src/Moryx.Runtime.Endpoints/Logging/Endpoint/ILogController.cs [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+using Moryx.Runtime.Endpoints.Logging.Endpoint.Models;
+using Moryx.Runtime.Endpoints.Logging.Endpoint.Request;
+using System.Collections.Generic;
+
+namespace Moryx.Runtime.Endpoints.Logging.Endpoint
+{
+    /// <summary>
+    /// Service contract for logging features of the maintenance.
+    /// </summary>
+    internal interface ILogController
```
Comment (Toxantron) [inline src/Moryx.Runtime.Endpoints/Logging/Endpoint/ILogController.cs:14]: That interface is not relevant anymore.

File: src/Moryx/Logging/ILoggingComponent.cs [hunk 2]
```diff
@@ -8,9 +8,5 @@ namespace Moryx.Logging
     /// </summary>
     public interface ILoggingComponent
     {
-        /// <summary>
-        /// Logger of this component
-        /// </summary>
-        IModuleLogger Logger { get; set; }
```
Comment (Toxantron) [inline src/Moryx/Logging/ILoggingComponent.cs]: We can't remove a property from a released interface in a minor.

File: src/Moryx.Runtime.Endpoints/Logging/Endpoint/LogController.cs [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+                AppenderId = _loggingAppender.AddRemoteLogAppender(request.Name, request.MinLevel)
+            };
+        }
+
+        [HttpGet("appender/{appenderId}")]
+        public ActionResult<IEnumerable<LogMessageModel>> GetMessages([FromRoute] string appenderId)
+        {
+            var appender = int.Parse(appenderId);
+            if (!_loggingAppender.ValidAppender(appender))
+                throw new ArgumentException($"Appender with Id {appenderId} could not be found");
```
Comment (Toxantron) [inline src/Moryx.Runtime.Endpoints/Logging/Endpoint/LogController.cs:63]: For an unknown id I would prefer a NotFound instead of InternalServerError 500

**Review by Toxantron — COMMENTED** [review]
I know you we had to remove ParallelOperations, but we need some replacement to prevent once created `RemoteAppenders` from flooding the memory if no one flushes the messages or removes the appenders. I'd say every log appender that has not been checked for more than 30s can be considered dead.

**Comment by milmilkat** [comment]
@Toxantron I am not sure if I could understand you correctly, but take a look and let me know if further ammendents are required.

**Review by Toxantron — APPROVED** [review]

**Pull request merged**